lib/oli/publishing.ex

```diff
@@ -392,39 +392,43 @@ defmodule Oli.Publishing do
       []
   """
   def retrieve_visible_sources(user, institution) do
```

Comment:
The more "Elixir" way to handle user of `nil` is to simply add a pattern match impl and leave the original impl untouched: 

`def retrieve_visible_sources(nil, _), do: []`